pyfar/classes/audio.py

```diff
@@ -166,7 +166,11 @@ def __getitem__(self, key):
 
 
         """
+        length = self._data.shape[-1]
```

Comment:
```suggestion
        length = self._data.shape[-1]
        if hasattr(key, '__iter__'):
            key = (*key, slice(None, None, None))
```

Reply:
I had a working solution already, but I had to update the tests, which is now done.

Reply:
it does not fix the problem:
```
import pyfar as pf
signal = pf.Signal([[[0, 0, 0, 0, 0], [1, 1, 1, 1, 1]]], 1)
signal.cshape
signal[..., 1]
```
``signal[..., 1]``still does not work

Reply:
therefore I added the ``slice(None, None, None)``, then signal[..., 1] would return the correct data signal[:, 1]

Reply:
Ah, now I got what you ment. Should be fine now.